app/src/main/res/xml/home_scene.xml

```diff
@@ -12,6 +12,7 @@
         <OnSwipe
             motion:dragDirection="dragUp"
             motion:touchAnchorId="@id/home_component"
+            motion:touchRegionId="@id/home_component"
```

Comment:
:stars: new feature :stars: 

Now the motion layout only activates if the touch is within the region :100: